tests/test_auth.py

```diff
@@ -49,9 +49,13 @@ def test_003_init_secondary():
     assert node2.wait_until_state(target_state="secondary")
     assert node1.wait_until_state(target_state="primary")
 
+    eq_(node1.get_synchronous_standby_names_local(), '*')
```

Comment:
To better check for the found issue, this seems like a better test:
```suggestion
    assert node1.wait_until_state(target_state="primary")
    eq_(node1.get_synchronous_standby_names_local(), '*')
    assert node2.wait_until_state(target_state="secondary")
```

Same for the other places where you added this new check after `wait_until_state` calls.

Reply:
Fair, done that way now.